libavif-sys/Cargo.toml

```diff
@@ -15,14 +15,15 @@ readme = "README.md"
 default = ["codec-dav1d", "codec-rav1e"]
 codec-rav1e = ["rav1e","nasm-rs"]
 codec-dav1d = []
-codec-aom = []
+codec-aom = ["libaom-sys"]
 
 __internal_aom_generic_target = []
```

Comment:
We need to pass this to libaom-sys to make it build on docs.rs

Alternatively we could check for docs.rs by using the recently introduced [`DOCS_RS` environment variable](https://github.com/rust-lang/docs.rs/pull/852), which would allow us to remove this feature from all of the crates

Reply:
I've switched to the env var